### PR TITLE
Full interactive feature

### PR DESCRIPTION
**Pull request by arabiu033**
- Add a new file to handle the status of the task
- Update the index.js and added some imports
- Update the task.js file to work with new changes
- Add some new resources
- Fix linters

### CODE REVIEW

File: src/status.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  for (let i = 0; i < task.tasks.length; i += 1) {
+    task.tasks[i].index = i + 1;
+  }
+  localStorage.setItem('listOfTasks', JSON.stringify(task.tasks));
+  ul.innerHTML = '';
+  task.populate();
+  updateCheck(task);
+};
+
+export { updateCheck, clearTasks };
```
Comment (rotimiazeez) [inline src/status.js:36]: - Please note that when a new task is added to the list it does not appear on the screen. Kindly fix it 👍🏽
[Screencast from 29-09-2022 16:37:09.webm](https://user-images.githubusercontent.com/44624138/193088061-982c1987-1015-414f-a461-959040734a63.webm)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @arabiu033 👏🏽, 
## Good job so far! 👍🏽 
### Highlights ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯
- Nice UI 👍🏽  
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/status.js (hunk 1 above)
Comment (arabiu033) [reply src/status.js:36]: Tasks are added using the Arrow Key.
Let me add the keyboard support then. thanks

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @arabiu033 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ Project is setup correctly
 ✔️ All linters are passing
✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**